tools/dep_updaters/update-acorn-walk.sh

```diff
@@ -7,10 +7,12 @@
 
 set -ex
 
+BASE_DIR=$(cd "$(dirname "$0")/../.." && pwd)
```

Comment:
It seems that `BASE_DIR` and `ROOT` are the same thing. Use `ROOT`?

Reply:
Yes copy paste mistake